mtp_cashbook/apps/disbursements/forms.py

```diff
@@ -177,12 +177,13 @@ class SendingMethodForm(DisbursementForm):
 
 
 class RecipientContactForm(DisbursementForm):
-    recipient_name = forms.CharField(label='Their name')
+    recipient_name = forms.CharField(label='Their first name')
+    recipient_last_name = forms.CharField(label='Last name')
     address_line1 = forms.CharField(label='Their address')
     address_line2 = forms.CharField(required=False)
-    city = forms.CharField(required=False)
-    postcode = forms.CharField(label='Their postcode')
-    email = forms.EmailField(label='Their email address (if provided)', required=False)
+    city = forms.CharField(label='Town or city', required=False)
+    postcode = forms.CharField(label='Postcode')
```

Comment:
Did Jane specifically want "Their postcode" to be "Postcode"?

Reply:
We either put "their" in front of every box or none of them, looks v weird to me right now

Reply:
Those are kind of sub-fields under the header of "Their address" though. I mean if/wehn we add country I think it will be confusing if it says "Their country" because that could mean a variety of things, whereas just having "Country" under "Their address" is clearer.